help/en/html/hardware/can/cbus/Names.shtml

```diff
@@ -158,14 +158,45 @@
       
       <p>See <a href="../../../doc/Technical/TurnoutFeedback.shtml">JMRI : Turnout Feedback</a> for more info.</p>
       
+      
+      <h4><a name="reporters" id="reporters">Reporter ( RFID ) data from CBUS</a></h4>
+      <p><a href="../../../tools/Reporters.shtml">JMRI Reporters</a> do not have Off or On events, they just use a device ( short event ) or node number.</p>
+      <p>A typical system name for a reporter would be <code>MR123</code> or <code>MR1234</code> ( no event On or Off ) .</p>
+      <p>The DDES and ACDAT OPC's are used for reporter data.</p>
+      <p>When a DDES or ACDAT OPC is heard on the network, 
+      JMRI will look for a reporter matching the device or node number in the 
+      <a href="../../../../package/jmri/jmrit/beantable/ReporterTable.shtml">Reporter Table</a>.</p>
+      
+      <p>If a reporter exists, the <a href="../../../tools/IdTags.shtml">ID tag</a> 
+      within the 5 data bytes will be looked up from the 
+      <a href="../../../../package/jmri/jmrit/beantable/IdTagTable.shtml">ID Tag table</a>.</p>
+      
+      <p>If there's no matching ID Tag on the table, one will be created and updated.</p>
+      
+      <p>If the ID tag was previously active for another reporter, 
+      the previous reporter will have the tag removed from its report.</p>
+      
+      <p>If the DDES or ACDAT message is coming FROM JMRI, the previous reporter will not be updated,
+      the existing reporeter is only cleared by messages coming in to JMRI. </p>
+      <p>Reporters are created automatically and will populate the Reporter table when 
+      a DDES or ACDAT OPC is heard on the network.</p>
+      <p>Valid reporter numbers are minimum 1, maximum 65535.</p>
+      <p>The DDES ( device number ) and ACDAT ( node number ) messages are 
+      currently handled in exactly the same way, ie the first 2 bytes are used as the Reporter identifier.</p>
+      <p>This means that a reporter created via a DDES of device 77 will respond to a ACDAT of node 77,
+      and vice versa.</p>
+      
+      <p>Reporters are saved in your main panel file, along with turnouts and sensors etc.</p>
+      <p>ID Tags cross-session automatically, no saving is necessary.</p>
       </div>
       
       <h2><a name="sysname" id="sysname">System Names</a></h2>
        <div>
         <p>When adding an item to your JMRI
         <a href="../../../../package/jmri/jmrit/beantable/SensorTable.shtml">Sensors Table</a>,
-        <a href="../../../../package/jmri/jmrit/beantable/TurnoutTable.shtml">Turnouts Table</a> or
-        <a href="../../../../package/jmri/jmrit/beantable/LightTable.shtml">Lights Table</a>,
+        <a href="../../../../package/jmri/jmrit/beantable/TurnoutTable.shtml">Turnouts Table</a>,
```

Comment:
This is not the name of these tables. Should be referred to as (singular) ``Turnout Table`` and ``Light Table``.